### ISSUE TITLE
Certbot and apache configuration do not align

### ISSUE THREAD

**Issue by nicolasberens** [freifunk-berlin/berlin.freifunk.net#142]
certbot puts the challenge into  /var/www/404

(/etc/letsencrypt/renewal/berlin.freifunk.net.conf)

the apache configuration needs a location block that will use it as a document root for requests to .well-known, otherwise the verification will fail.

please test afterwards with `certbot renew --dry-run`

**Comment by sarumpaet** [comment]
```
Your certificate (or certificates) for the names listed below will expire in 16 days (on 2024-06-21). Please make
sure to renew your certificate before then, or visitors to your web site will encounter errors.
berlin.freifunk.net
www.berlin.freifunk.net
```
@noxilixon

**Comment by noxilixon** [comment]
I was super busy with other staff, but will focus on that issue again next week. The current certificate is valid until `Fri 21 Jun`. I will try to switch to caddy (as discussed in matrix) before that time. Or I will once again do a manual renewal.

**Comment by noxilixon** [comment]
This was fixed in freifunk-berlin/ansible#122
